src/components/Books.js

```diff
@@ -9,3 +9,5 @@ const Books = () => (
 );
 
 export default Books;
+
+// ZMP8Cx3pL0HB31pM9poX
```

Comment:
Great job @AhmedNazirMusah 🤝

- [ ] [Optional] You may always remove commented code from your project as this is regarded as a bad practice.

Reply:
Thanks for drawing my attention to that, thank you 😃 .